modules/ROOT/pages/audit-logging.adoc

```diff
@@ -0,0 +1,63 @@
+= Audit Logging
+:page-edition: Enterprise
+:description: Audit Logging provides tools for administrators to track operational irregularities and to support regulatory and security compliance standards, such as link:https://www.hhs.gov/hipaa/index.html[HIPAA] and link:https://soc2.co.uk/soc2[SOC-2].
+
+[abstract]
+{description}
+
+== About Audit Logging
+
+Key concepts, use cases and sign posting
+Explain audit log files, only one active at a time that they’re written etc
+
+Audit Logging provides tools for administrators to track operational irregularities and to support regulatory and security compliance standards, such as link:https://www.hhs.gov/hipaa/index.html[HIPAA] and link:https://soc2.co.uk/soc2[SOC-2].
+
+There is only one audit log file active at a time, which is periodically rotated based on the user-configured `rotateInterval` or `rotateSize` properties.
+Logs are immutable and only accessible via read-only APIs, thus ensuring logs cannot be altered once recorded.
+Viewing and managing audit logs permissions are enforced through Sync Gateway RBAC controls. 
+Logs are encrypted both in transit to prevent unauthorized access or tampering, all communication supports TLS.
```

Comment:
There's no transport of sync gateway logs, they are written locally on each Sync Gateway node. *There is transport in app services.

Reply:
Changed, thanks!